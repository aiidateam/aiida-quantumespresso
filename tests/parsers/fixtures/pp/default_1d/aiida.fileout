        0.0000000000      -11.3601206194
        0.0101010101      -11.2478484712
        0.0202020202      -10.9031326566
        0.0303030303      -10.3221343782
        0.0404040404       -9.5416598550
        0.0505050505       -8.6478189645
        0.0606060606       -7.7382726743
        0.0707070707       -6.8674698320
        0.0808080808       -6.0247906850
        0.0909090909       -5.1668428440
