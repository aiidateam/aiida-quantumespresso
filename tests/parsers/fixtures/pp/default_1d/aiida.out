
     Program POST-PROC v.6.4 starts on 23Oct2019 at  8:53: 0 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
         "P. Giannozzi et al., J. Phys.:Condens. Matter 29 465901 (2017);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI), running on     4 processors

     MPI processes distributed on     1 nodes
     R & G space division:  proc/nbgrp/npool/nimage =       4

     IMPORTANT: XC functional enforced from input :
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)
     Any further DFT definition will be discarded
     Please, verify this is what you really want

               file C.pbe-n-kjpaw_psl.1.0.0.UPF: wavefunction(s)  2S 2P renormalized

     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min         326     164     40                 8876     3132     386
     Max         327     165     41                 8881     3135     389
     Sum        1305     657    161                35513    12533    1551


     Check: negative core charge=   -0.000013

     Calling punch_plot, plot_num =  11
     Writing data to file  aiida.filplot
     Reading data from file  aiida.filplot

     Writing data to be plotted to file aiida.fileout
     Min, Max, imaginary charge:   -11.360121    1.003621    0.000000
     Plot Type: 1D along a line        Output format: gnuplot      

     POST-PROC    :      0.51s CPU      0.57s WALL


   This run was terminated on:   8:53: 0  23Oct2019            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
